ansible/group_vars/reference/resources.yml

```
@@ -123,3 +123,11 @@ COMMON_CARTRIDGE_CPU_LIMITS: "1000m"
 COMMON_CARTRIDGE_MEMORY_LIMITS: "2Gi"
 COMMON_CARTRIDGE_CPU_REQUESTS: "100m"
 COMMON_CARTRIDGE_MEMORY_REQUESTS: "2Gi"
+TLDRAW_SERVER_CPU_LIMITS: "2000m"
+TLDRAW_SERVER_MEMORY_LIMITS: "2Gi"
+TLDRAW_SERVER_CPU_REQUESTS: "100m"
+TLDRAW_SERVER_MEMORY_REQUESTS: "150Mi"
+TLDRAW_SERVER_WORKER_CPU_LIMITS: "2000m"
+TLDRAW_SERVER_WORKER_MEMORY_LIMITS,: "2Gi"
+TLDRAW_SERVER_WORKER_CPU_REQUESTS: "100m"
+TLDRAW_SERVER_WORKER_MEMORY_REQUESTS,: "150Mi"
```

Littel bit sorting and for memory request = limit.
CPU Time kann be overbooked but memory shoud not be overbooked.
```suggestion
TLDRAW_SERVER_CPU_LIMITS: "2000m"
TLDRAW_SERVER_CPU_REQUESTS: "100m"
TLDRAW_SERVER_MEMORY_LIMITS: "2Gi"
TLDRAW_SERVER_MEMORY_REQUESTS: "2Gi"
TLDRAW_SERVER_WORKER_CPU_LIMITS: "2000m"
TLDRAW_SERVER_WORKER_CPU_REQUESTS: "100m"
TLDRAW_SERVER_WORKER_MEMORY_LIMITS,: "2Gi"
TLDRAW_SERVER_WORKER_MEMORY_REQUESTS,: "2Gi"
```